fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_objectives.dm

```diff
@@ -37,6 +33,10 @@
 /datum/objective/bloodsucker/lair
 	name = "claimlair"
 
+/datum/objective/bloodsucker/lair/New()
+	update_explanation_text()
```

Comment:
Is there any reason why you cant add this to /datum/objective/bloodsucker/New() instead of adding it to all objectives individually?

Reply:
get real